src/Page/Polls.elm

```diff
@@ -11,85 +12,143 @@ import Cmd exposing (withCmd, withNoCmd)
 import Route
 import Session exposing (Session, Viewer)
 import Task
 import Task.Extra
+import Time
 
 
-type Message
-    = GotNewPolls (List Poll)
-    | RequestPolls
-    | DeletePoll Poll
+
+-- MODEL
 
 
 type alias Model =
     { viewer : Viewer
     , polls : List Poll
+    , order : Sorting.Order
     }
 
 
 init : Viewer -> ( Model, Cmd Message )
 init viewer =
     { viewer = viewer
     , polls = []
+    , order = Sorting.TitleAsc
     }
-        |> withCmd [ Task.perform identity <| Task.succeed RequestPolls ]
+        |> withCmd [ Cmd.succeed NowRequestPolls ]
+
+
+
+-- MESSAGE
+
+
+type Message
+    = GotAllPolls (List Poll)
+    | NowRequestPolls
+    | NowDeletePoll Poll
+    | NowSetOrder Sorting.Order
 
 
 update : Message -> Model -> ( Model, Cmd Message )
 update message model =
     case message of
-        GotNewPolls polls ->
+        GotAllPolls polls ->
             { model | polls = polls } |> withNoCmd
 
-        RequestPolls ->
+        NowRequestPolls ->
             model
                 |> withCmd
                     [ Api.Polls.getAllPolls (Session.viewerCredentials model.viewer) identity
                         |> Task.mapError (always [])
                         |> Task.Extra.execute
-                        |> Cmd.map GotNewPolls
+                        |> Cmd.map GotAllPolls
                     ]
 
-        DeletePoll poll ->
+        NowDeletePoll poll ->
             model
                 |> withCmd
-                    [ Api.Polls.delete (Session.viewerCredentials model.viewer) poll RequestPolls
-                        |> Task.mapError (always RequestPolls)
+                    [ Api.Polls.delete (Session.viewerCredentials model.viewer) poll NowRequestPolls
+                        |> Task.mapError (always NowRequestPolls)
                         |> Task.Extra.execute
                     ]
 
+        NowSetOrder order ->
+            { model | order = order } |> withNoCmd
+
+
+{-| Request polls refresh every 10 seconds.
+-}
+subscriptions : Model -> Sub Message
+subscriptions _ =
+    Time.every (10 * 1000) (always NowRequestPolls)
 
+
+
+-- VIEW
+
+
+{-| The main view of the page. Displays the list of polls, as well as the floating action button
+that navigates to the poll creation screen.
+-}
 view : Model -> List (Html Message)
 view model =
-    [ table model.polls ]
-        ++ [ Picasso.FloatingButton.a
-                [ class "fixed right-0 bottom-0 m-8"
-                , Route.href Route.NewPoll
-                ]
-                [ img [ src "icon/action-button-plus.svg" ] []
-                , div [ class "ml-4" ] [ text "New poll" ]
-                ]
-           ]
+    let
+        sorted =
+            List.sortWith (Sorting.ordering model.order) model.polls
+    in
+    [ viewTable model.order sorted ] ++ [ fab ]
 
 
-table : List Poll -> Html Message
-table polls =
+{-| The floating action button that enables navigation to the new poll screen.
+-}
+fab : Html Message
```

Comment:
Shortness or clarity? floatingActionButton seems better to me.

Reply:
Probably a bad habit of Android development, where the acronym is used very often.